examples/high_frequency/emc/busbar.py

```diff
@@ -117,18 +117,56 @@
 print(q3d.net_sources("Bar2"))
 print(q3d.net_sources("Bar3"))
 
+# Create Matrix Reduction Operations
+
+# Series of Bar1 and Bar2
+
+mr_series = q3d.insert_reduced_matrix(
+    operation_name="JoinSeries",
+    assignment=["Sink1", "Source2"],
+    reduced_matrix="MR_1_Series",
+    new_net_name="Series1",
+)
+
+# Add Parallel with Bar3
+
+# mr_series = [b for b in q3d.matrices if b.name == 'MR_1_Series'] [0]
+mr_series.add_operation(
+    operation_type="JoinParallel",
+    source_names=["Source1", "Source3"],
+    new_net_name="SeriesPar",
+    new_source_name="src_par",
+    new_sink_name="snk_par",
+)
+
+# Series of Bar1 and Bar2
+
+mr_series2 = q3d.insert_reduced_matrix(
+    operation_name="JoinSeries",
+    assignment=["Sink1", "Source2"],
+    reduced_matrix="MR_2_series",
+    new_net_name="Series2",
+)
+
+# Add Series with Bar3
+mr_series2.add_operation(
+    operation_type="JoinSeries",
+    source_names=["Sink3", "Source1"],
+    new_net_name="MR_2_Series1",
+)
+
 # Create the solution setup and define the frequency range for the solution.
 
 setup1 = q3d.create_setup(props={"AdaptiveFreq": "100MHz"})
 sweep = setup1.add_sweep()
-sweep.props["RangeStart"] = "1MHz"
-sweep.props["RangeEnd"] = "100MHz"
+sweep.props["RangeStart"] = "0MHz"
+sweep.props["RangeEnd"] = "1000MHz"
 sweep.props["RangeStep"] = "5MHz"
 sweep.update()
 
 # ### Set up for postprocessing
```

Comment:
```suggestion
# ## Set up for postprocessing
```
@tizianrot be consistent with the other headers.

Reply:
@tizianrot the title of each code section, in this case "Set up for postprocessing", begins with 3 hashtag, so please apply @gmalinve suggestion